code/modules/events/bureaucratic_error.dm

```diff
@@ -0,0 +1,16 @@
+/datum/round_event_control/bureaucratic_error
+	name = "Bureaucratic Error"
+	typepath = /datum/round_event/bureaucratic_error
+	max_occurrences = 1
+	weight = 5
+
+/datum/round_event/bureaucratic_error
+	announceWhen = 1
+
+/datum/round_event/bureaucratic_error/announce(fake)
+	priority_announce("A recent bureaucratic error in the Organic Resources Department may result in personnel shortages in some departments and redundant staffing in others.", "Paperwork Mishap Alert")
```

Comment:
Append "This issue should be resolved shortly." even though the event is permanent.